required/tools/array.dtx

```diff
@@ -1432,10 +1458,15 @@ Bug reports can be opened (category \texttt{#1}) at\\%
 %    \begin{macrocode}
 \protected\def\textonly@unskip{\ifhmode\unskip\fi}
 %    \end{macrocode}
+% \changes{v2.6g}{2024/10/12}{Further work to support optional args in preamble (gh/1468)}
+%  We need an engine-protected function that is just \cs{@firstofone}:
+%    \begin{macrocode}
+\protected\def\@protected@firstofone#1{#1}
```

Comment:
Should be `\long` (though the usage here isn't requiring that).

Reply:
Added, see 7db95fb6 (Add a missing \long, 2024-10-12) by Joseph.